### PR TITLE
remove clickedArtworkGroup helpers

### PR DESCRIPTION
**Pull request by pvinis**
Removing the use of clickedArtworkGroup helpers, after the cohesion RFC to remove helpers.

Sibling PR https://github.com/artsy/cohesion/pull/236

### CODE REVIEW

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx [hunk 1]
```diff
@@ -9,9 +9,11 @@ import { AuctionArtworksRailPlaceholder } from "../AuctionArtworksRailPlaceholde
 import { tabTypeToContextModuleMap } from "../../Utils/tabTypeToContextModuleMap"
 import { useTracking } from "react-tracking"
 import {
-  clickedArtworkGroupHeader,
-  ClickedArtworkGroupHeaderArgs,
+  ActionType,
+  ClickedArtworkGroup,
```
Comment (damassi) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:13]: Is it ok that the helper types are still here? This is pretty confusing

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx [hunk 2]
```diff
@@ -113,3 +112,20 @@ export const AuctionArtworksRailFragmentContainer = createFragmentContainer(
...
   }
 )
+
+const tracks = {
+  clickedArtworkGroupHeader: (
+    contextModule: ContextModule,
+    contextPageOwnerType: PageOwnerType,
+    saleID: string,
+    saleSlug: string
+  ): ClickedArtworkGroup => ({
```
Comment (damassi) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:122]: this should either be removed or defined in force... or?

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx (hunk 1 above)
Comment (pvinis) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:13]: That's the schema type. small `c` was the helper. The schema types make sense to be there, to verify we send the right things.

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx (hunk 2 above)
Comment (pvinis) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:122]: You mean the schema type? That's here https://github.com/artsy/cohesion/blob/d9f09f800282a372e2e182a407b42ee153181491/src/Schema/Events/Click.ts#L164.

**Review by mdole — APPROVED** [review]
LGTM!

**Comment by mdole** [comment]
Oh, although...we should probably introduce regression tests in the same pattern as with this Eigen PR, right? https://github.com/artsy/eigen/pull/5492

**Pull request merged**